### PR TITLE
[Win32] - use formed packages for all depends

### PR DESCRIPTION
**Pull request by Memphiz**
As already announced - i repackaged all win32 depends to use formed packages. I also included the commit from @Karlson2k for allowing to redownload the packages if download failed.

Worked fine as of jenkins:

http://jenkins.kodi.tv/job/WIN-32/3342/

This hopefully works around any possible network issues on the windows slave.

### CODE REVIEW

**Comment by Memphiz** [comment]
wow - is this a feature of github now? comments are visible in the pr conversation - kinda cool.

**Comment by Karlson2k** [comment]
Not all comments are visible:
https://github.com/Memphiz/xbmc/commit/6b634f9f0d060b6d416619a8277227ae2d91a5f3#commitcomment-9446824

**Comment by Memphiz** [comment]
@Karlson2k updated - and testbuilt:

http://jenkins.kodi.tv/job/WIN-32/3346/

**Comment by Karlson2k** [comment]
@Memphiz Nice, thanks!
Minor and mostly cosmetics: we was trying to keep `0_package.list` sorted alphabetically.

**Comment by Memphiz** [comment]
yes and now you failed ;) *just kidding - will do*

**Comment by Memphiz** [comment]
@Karlson2k done

**Comment by Karlson2k** [comment]
@Memphiz Good!
Squash last commit with first one and it will be ready to merge.

**Comment by Memphiz** [comment]
I have better things to do instead of squashing down in a non-logical way. We always seperate cosmetics with functional changes everywhere else - so i decided to do the same here - sorry mate.

File: project/BuildDependencies/scripts/0_package.list [hunk 1]
```diff
@@ -5,28 +5,48 @@
...
+libfribidi-0.19.2-win32.7z
 libiconv-1.14-win32-vc120.7z
 libjpeg-turbo-1.2.0-win32.7z
+liblzo-2.04-win32.7z
+libmicrohttpd-0.4.5-win32-1.7z
 libnfs-1.6.2-win32.7z
+libogg-vc100-1.2.0-win32.7z
+libplist-1.7-win32-3.7z
+libpng-1.5.13-win32.7z
+librtmp-20110723-git-b627335-win32-1.7z
```
Comment (FernetMenta) [inline project/BuildDependencies/scripts/0_package.list]: you know that this is the wrong file, don't you? (-1 at end) nevertheless does not matter, I will update with new lib anyway.

Comment (MartijnKaijser) [inline project/BuildDependencies/scripts/0_package.list]: that' s the correct file as he repackaged it

**Comment by Karlson2k** [comment]
@Memphiz Just noticed that list was already unsorted so I agree with this way. :smile:

**Pull request merged**